django/applications/catmaid/control/cropping.py

```diff
@@ -179,70 +222,9 @@ def to_z_index(z, stack, zoom_level, enforce_bounds=True) -> int:
         section = min(max(section, 0.0), stack.dimension.z - 1.0)
     return int(section)
 
-def addMetaData(path:str, job, result) -> None:
```

Comment:
Now that we're using PIL for the image manipulations, we don't need to save, open, read, save again to add metadata. The remainder is just mangling `CropJob` attributes, so I put them in a method of that class.

Reply:
That's certainly a better place and good that we could remove this re-opening part.